### PR TITLE
Fix to some smaller bugs in the compliant Logger module

### PR DESCRIPTION
**Pull request by felixnext**
Some small bug fixes:

## Changes

- Fixed bug where None values in dict could break the metric function /
- Fixed format commands on f-strings /
- Added additional exception handling to the library import (in case library is broken)

## Future

- You may show was is explicitly _not_ in the scope of this pull request.

## Key files

- logger.py
- system_info.py

## To-do

- [ ] If you have any action items which can't easily be expressed as comments
  on the pull request, or need to be carried out after the pull request is
  completed, express them as Markdown checkboxes.

### CODE REVIEW

File: shrike/compliant_logging/logging.py [hunk 1]
```diff
@@ -350,7 +350,8 @@ def metric(
                         row_str = f"TableMetric     | {i:05}"
                         for key in value:
                             col = value[key]
-                            row_str += f" | {(col[i] if i < len(col) else ''):15}"
+                            col = col[i] if i < len(col) and col[i] is not None else ""
```
Comment (dkmiller) [inline shrike/compliant_logging/logging.py:353]: You can just put `col[i] if i < len(col) and col[i] else ""`, Python's boolean coercion treats `col[i]` and `col[i] is not None` the same.

**Review by dkmiller — CHANGES_REQUESTED** [review]
You'll need to do a patch version bump as well.

**Review by haozhestat — COMMENTED** [review]
Could you update the unit tests as well

**Review by dkmiller — APPROVED** [review]
Thanks for contributing!

**Comment by felixnext** [comment]
@dkmiller @haozhestat  I do not think that I have access to the PyPi deployment project. Would one of you do that and complete the merge?

**Pull request merged**